src/components/navigation/nav.module.scss

```diff
@@ -0,0 +1,317 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,400i,500,500i,700');
+@import '../../../styles/variables.scss';
+@import '../../../styles/mixins.scss';
+
+
+.navbar {
+  font-family: $interRegular;
+  font-size: 14px;
+  color: $black;
+  box-sizing: border-box;
+} //global body styles
+
+.navbar *{
+  
+  box-sizing: inherit;
+}//global box-sizing
+
+::-webkit-scrollbar {
+  display: none;
+  }
+  i{
+    font-size: 18px;
+    cursor: pointer;
+  }
+
+.Link{
+  color: $black
+} 
+ 
+a {
+  text-decoration: none;
+  cursor: pointer;
+} //global link styles
+.vectors{
+  display:flex;
+  align-items: center;
+}
+.right_nav{
+  nav{
+    display: none;
+  }
+}
+.top__bar {
+  z-index: 20;
+  position: relative;
+  width: 100%;
+  background-color: $white;
+  height: 48px;
+  border-bottom: $borderColor;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0px 10px;
+
+  .logo__icon {
+    width: 10%;
+  }
+  .logo__text { 
+    height: 18px;
+  }
+  nav{
+      ul{
+         list-style: none;
+      }
+
+      li{
+          display: inline-block;
+          padding: 10px 40px;
+          transition: all 0.4s ease 0s;
```

Comment:
this transition is also in mixin,  check there, and all transitions have to be the same